yktrace/src/errors.rs

```diff
@@ -1,33 +1,27 @@
-use crate::debug;
 use std::fmt::{self, Display, Formatter};
-use ykpack::DefId;
 
 #[derive(Debug)]
 /// Reasons that a trace can be invalidated.
 pub enum InvalidTraceError {
-    /// There is no SIR for the DefId of a location in the trace.
-    /// The second element is the definition path string.
-    NoSir(DefId, String),
+    /// There is no SIR for the location in the trace.
+    /// The string inside is FIXME
```

Comment:
Is this something to be fixed in the future?

Reply:
The FIXME should be fixed now. Thanks for pointing this out.